js/hitbtc.js

```diff
@@ -805,6 +804,37 @@ module.exports = class hitbtc extends Exchange {
         let response = await this.tradingGetOrdersRecent (this.extend (request, params));
         return this.parseOrders (response['orders'], market, since, limit);
     }
+    
+    async fetchOrderTrades (id, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let response = await this.tradingGetTradesByOrder (this.extend ({
+            'clientOrderId': id,
+        }, params));
+        return this.parseOrderTrades (response['trades'], since, limit);
+    }
+    
+    parseOrderTrades (trades, since = undefined, limit = undefined) {
+        let result = Object.values (trades).map (trade => this.parseOrderTrade (trade));
+        return this.filterBySinceLimit (result, since, limit);
+    }
+    
+    parseOrderTrade (trade) {
+        let market = this.markets_by_id[trade['symbol']];
+        return {
+            'info': trade,
+            'id': trade['tradeId'],
+            'order': trade['clientOrderId'],
+            'timestamp': trade['timestamp'],
+            'datetime': this.iso8601 (trade['timestamp']),
+            'symbol': market['symbol'],
+            'type': undefined,
+            'side': trade['side'],
+            'price': parseFloat (trade['execPrice']),
+            'amount': parseFloat (trade['execQuantity']*market['lot']),
```

Comment:
Need spaces around the multiplication operator ` * ` here, otherwise Python will complain... same for the next line.

Reply:
Will update